core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/NativeImageBuildStep.java

```diff
@@ -694,12 +705,19 @@ public NativeImageInvokerInfo build() {
                     nativeImageArgs.add("-H:-ParseOnce");
 
                     // AdditionalSecurityProviders
-                    if (nativeImageSecurityProviders != null && !nativeImageSecurityProviders.isEmpty()) {
+                    if (!nativeImageSecurityProviders.isEmpty()) {
```

Comment:
Was this meant to be included in this PR? Doesn't seem related

Reply:
Yeah, the null check is not needed. I can remove this change if you like

Reply:
Removed